manifests/nodejs.yml

```diff
@@ -61,6 +61,8 @@ refs:
   - &ref_5_52_0 '>=5.52.0'
   - &ref_5_54_0 '>=5.54.0'
   - &ref_5_55_0 '>=5.55.0'
+  - &ref_5_56_0 '>=5.56.0'
```

Comment:
i think we can consider all features to start from 5.57 instead so it's merged and simpler